workshops/migrations/0060_published_help_text.py

```diff
@@ -0,0 +1,64 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+import django.core.validators
+import re
+import django_countries.fields
+
+
+def add_address_to_online_events(apps, schema_editor):
+    """Set Event.address if empty for online events."""
+    Event = apps.get_model('workshops', 'Event')
+    Tag = apps.get_model('workshops', 'Tag')
+    online = Tag.objects.get(name='online')  # should be created via 0040_*
+
+    Event.objects.filter(tags__in=[online]) \
+                 .filter(models.Q(address=None) | models.Q(address='')) \
+                 .update(address='Internet')
```

Comment:
On Fri, Nov 20, 2015 at 08:30:09AM -0800, Piotr Banaszkiewicz wrote:
> … there can be instances of events that may have a different
> location (like streamed events?).

Ah, this is a good reason to keep the ‘online’ tag (so we have a way
to mark streamed events).  However, events in the W3 country will
*always* be online, so I'd make sure we set that tag for such events
in Event.save() (you've set the rest of those fields in 90198e8, and
I'd suggest you want similar updates in this migration for all
existing W3 events).